public/javascripts/case-progress.js

```diff
@@ -35,11 +32,15 @@
     return noteEventListener
   }
 
-  const setupAutoSave = (textarea) => {
-    textarea.addEventListener('keypress', getAutoSaveHandler(textarea))
+  const setupAutoSaveHearingNote = (textarea) => {
+    textarea.addEventListener('keypress', getAutoSaveHandler(textarea,  'summary/auto-save-new-note', (textarea, event) => ({ note: event.srcElement.value, hearingId: textarea.dataset.hearingid }) ))
   }
 
-  document.querySelectorAll('.auto-save-text').forEach(setupAutoSave)
+  document.querySelectorAll('.auto-save-text').forEach(setupAutoSaveHearingNote)
```

Comment:
this will suffer from the same issue we discussed previously, where you're binding the same handler multiple times (depending on how many ```.auto-save-text``` elements there are. Could you try using event delegation again?

Reply:
This bit has been test, delivered and is in prod so I don't want to disturb it now unless we agree with wider team on QA and doing UAT again. This is not part of the current ticket, I just had to change the function name to distinguish from autosave comment